packages/vuepress-theme-odyssey/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@okta/vuepress-theme-odyssey",
+  "version": "0.6.0",
+  "description": "VuePress theme for Odyssey Documentation",
+  "author": "Okta, Inc.",
+  "license": "Apache-2.0",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint . --ext .js,.vue && stylelint ./*.scss",
+    "test": "yarn lint"
+  },
+  "devDependencies": {
+    "@okta/odyssey": "^0.6.0",
+    "@okta/odyssey-icons": "^0.6.0",
+    "@okta/prism-theme-odyssey": "^0.6.0",
+    "eslint-config-vuepress": "^2.2.0",
+    "html-loader": "^1.1.0",
+    "node-sass": "4.13.1",
+    "prismjs": "^1.14.0",
```

Comment:
Looks like we have some dependencies here that are not used based on the current state of this PR. Let's make sure those get added when these dependencies are actively used!